website/src/components/Tasks/Task.tsx

```diff
@@ -0,0 +1,20 @@
+import { CreateTask } from "./CreateTask";
+import { EvaluateTask } from "./EvaluateTask";
+import { TaskTypes } from "./TaskTypes";
+
+export const Task = ({ tasks, trigger, mutate, mainBgClasses }) => {
+  const task = tasks[0].task;
+
+  function taskTypeComponent(type) {
+    const category = TaskTypes.find((taskType) => taskType.type === type).category;
+
+    switch (category) {
+      case "create":
+        return <CreateTask tasks={tasks} trigger={trigger} mutate={mutate} mainBgClasses={mainBgClasses} />;
+      case "evaluate":
+        return <EvaluateTask tasks={tasks} trigger={trigger} mutate={mutate} mainBgClasses={mainBgClasses} />;
+    }
+  }
+
+  return taskTypeComponent(task.type);
+};
```

Comment:
nit:

This component adds an extra level of indirection that is not necessary.
I would argue that you can directly use `CreateTask` or `EvaluateTask` in the corresponding task page.

Reply:
The reason I make a unified component and also combined everything into one array is because I'm assuming we will have more tasks in future, so we will need to know how to handle any type, given the response from the backend.